lib/controllers/foods-controller.js

```diff
@@ -8,7 +8,7 @@ function index (req, res) {
 };
 
 function create (req, res) {
-  Food.create(req)
+  Food.create(req.body.food.name, req.body.food.calories)
```

Comment:
I think we might want to save them in an object. Like:

let food = {name: req.body.food.name,
                   calories: req.body.food.calories, etc.}